lib/IRGen/GenDecl.cpp

```diff
@@ -1388,6 +1397,14 @@ SILLinkage LinkEntity::getLinkage(ForDefinition_t forDefinition) const {
     if (auto *proto = dyn_cast<ProtocolDecl>(decl->getDeclContext()))
       decl = proto;
 
+    // Method descriptors for internal class initializers can be referenced
+    // from outside the module.
```

Comment:
Only when building a non-resilient module, though. Or is that coming later?

Reply:
Note that wee use this method override mechanism for generic classes too, not just resilient classes.